client/errors.go

```diff
@@ -0,0 +1,130 @@
+	// and a list of the failures on each field.
+	ErrorKindInvalidAuthData = "invalid-auth-data"
+	// ErrorKindTermsNotAccepted: deprecated, do not document
+	ErrorKindTermsNotAccepted = "terms-not-accepted"
+	// ErrorKindNoPaymentMethods: deprecated, do not document
+	ErrorKindNoPaymentMethods = "no-payment-methods"
+	// ErrorKindPaymentDeclined: deprecated, do not document
+	ErrorKindPaymentDeclined = "payment-declined"
+	// ErrorKindPasswordPolicy: provided password doesn’t meet
+	// system policy.
+	ErrorKindPasswordPolicy = "password-policy"
+
+	// ErrorKindSnapAlreadyInstalled: the requested snap is
+	// already installed.
+	ErrorKindSnapAlreadyInstalled = "snap-already-installed"
+	// ErrorKindSnapNotInstalled:  the requested snap is not installed.
+	ErrorKindSnapNotInstalled = "snap-not-installed"
+	// ErrorKindSnapNotFound: the requested snap couldn’t be found.
+	ErrorKindSnapNotFound = "snap-not-found"
+	// ErrorKindAppNotFound: the requested app couldn’t be found.
+	ErrorKindAppNotFound = "app-not-found"
+	// ErrorKindSnapLocal: the requested snap couldn’t be found in
+	// the store.
+	ErrorKindSnapLocal = "snap-local"
+	// ErrorKindSnapNeedsDevMode: the requested snap needs devmode
+	// to be installed.
+	ErrorKindSnapNeedsDevMode = "snap-needs-devmode"
+	// ErrorKindSnapNeedsClassic: the requested snap needs classic
+	// confinement to be installed.
+	ErrorKindSnapNeedsClassic = "snap-needs-classic"
+	// ErrorKindSnapNeedsClassicSystem: the requested snap can’t
+	// be installed on the current system.
+	ErrorKindSnapNeedsClassicSystem = "snap-needs-classic-system"
+	// ErrorKindSnapNotClassic: snap not compatible with classic mode.
+	ErrorKindSnapNotClassic = "snap-not-classic"
+	// ErrorKindNoUpdateAvailable: the requested snap does not
+	// have an update available.
+	ErrorKindNoUpdateAvailable = "snap-no-update-available"
+
+	// ErrorKindRevisionNotAvailable: no snap revision available
+	// as specified.
+	ErrorKindRevisionNotAvailable = "snap-revision-not-available"
+	// ErrorKindChannelNotAvailable: no snap revision on specified
+	// channel. The value of the error is a rich object with
+	// requested snap-name, action, channel, architecture, and
+	// actually available releases as list of
+	// {"architecture":... , "channel": ...} objects.
+	ErrorKindChannelNotAvailable = "snap-channel-not-available"
+	// ErrorKindArchitectureNotAvailable: no snap revision on
+	// specified architecture. Value has the same format as for
+	// snap-channel-not-available:
+	ErrorKindArchitectureNotAvailable = "snap-architecture-not-available"
+
+	// ErrorKindChangeConflict: the requested operation would
+	// conflict with currently ongoing change. This is a temporary
+	// error. The error value is an object with optional fields
+	// snap-name, change-kind of the ongoing change.
+	ErrorKindChangeConflict = "snap-change-conflict"
+
+	// ErrorKindNotSnap : the given snap or directory does not
```

Comment:
Odd spacing around the colon.
```suggestion
	// ErrorKindNotSnap: the given snap or directory does not
```